src/pages/public_registration/PublicPublicationInstance.tsx

```diff
@@ -1,23 +1,50 @@
 import React, { FC } from 'react';
-import LabelContentRow from '../../components/LabelContentRow';
-import { JournalPublicationInstance } from '../../types/publication_types/journalRegistration.types';
+import { Skeleton } from '@material-ui/lab';
 import { useTranslation } from 'react-i18next';
+import { Link } from '@material-ui/core';
+import { JournalPublicationInstance } from '../../types/publication_types/journalRegistration.types';
+import LabelContentRow from '../../components/LabelContentRow';
 import { DegreePublicationInstance } from '../../types/publication_types/degreeRegistration.types';
 import { ReportPublicationInstance } from '../../types/publication_types/reportRegistration.types';
 import { PagesMonograph } from '../../types/registration.types';
+import { JournalType } from '../../types/publicationFieldNames';
+import useSearchRegistrations from '../../utils/hooks/useSearchRegistrations';
 
 export const PublicPublicationInstanceJournal: FC<{ publicationInstance: JournalPublicationInstance }> = ({
   publicationInstance,
 }) => {
   const { t } = useTranslation('registration');
-  const { articleNumber, issue, pages, volume } = publicationInstance;
+  const { type, articleNumber, issue, pages, volume, corrigendumFor } = publicationInstance;
 
   return (
-    <LabelContentRow minimal label={`${t('common:details')}:`}>
-      {volume && `${t('references.volume')} ${volume}`}
-      {issue && `, ${t('references.issue')} ${issue}`}
-      {pages?.begin && pages?.end && `, ${t('references.pages')} ${pages.begin}-${pages.end}`}
-      {articleNumber && `, ${t('references.article_number')} ${articleNumber}`}
+    <>
+      <LabelContentRow minimal label={`${t('common:details')}:`}>
+        {volume && `${t('references.volume')} ${volume}`}
+        {issue && `, ${t('references.issue')} ${issue}`}
+        {pages?.begin && pages?.end && `, ${t('references.pages')} ${pages.begin}-${pages.end}`}
+        {articleNumber && `, ${t('references.article_number')} ${articleNumber}`}
+      </LabelContentRow>
+      {type === JournalType.CORRIGENDUM && <OriginalArticleInfo originalArticleId={corrigendumFor} />}
+    </>
+  );
+};
+
+export const OriginalArticleInfo: FC<{ originalArticleId: string }> = ({ originalArticleId }) => {
```

Comment:
hvis denne likevel eksporteres burde den ikke ligget i egen fil ?

Reply:
den skulle faktisk ikke vært eksportert i hele tatt 😬 
(det er riktignok andre exports også i fila har, men lar det ligge for nå)